tests/integration/conftest.py

```diff
@@ -10,6 +10,10 @@
 
 
 def gds_version(driver: neo4j.Driver) -> str:
+    with driver.session() as session:
+        data = session.run("CALL gds.debug.sysInfo()").to_df()
+        print(data)
```

Comment:
@ali-ince Do you want to keep this print still?

Reply:
Yes please, because it took quite a bit of time to figure out that the GDS license on CI was expired :)